src/renderers/layouts/vertical.layout.ts

```diff
@@ -2,29 +2,49 @@ import { UISchemaElement, VerticalLayout } from '../../models/uischema';
 import { JsonFormsHolder} from '../../core';
 import {Renderer} from '../../core/renderer';
 import { JsonFormsRenderer } from '../renderer.util';
-import {Runtime} from '../../core/runtime';
+import {Runtime, RUNTIME_TYPE} from '../../core/runtime';
 
+export const VerticalLayoutRendererTester = (uischema: UISchemaElement) =>
+  uischema !== undefined && uischema !== null && uischema.type === 'VerticalLayout' ? 1 : -1;
 @JsonFormsRenderer({
   selector: 'jsonforms-verticallayout',
-  tester: (uischema: UISchemaElement) => uischema.type === 'VerticalLayout' ? 1 : -1
+  tester: VerticalLayoutRendererTester
 })
-class VerticalLayoutRenderer extends Renderer {
+export class VerticalLayoutRenderer extends Renderer {
 
   constructor() {
     super();
   }
   render(): HTMLElement {
     const div = document.createElement('div');
     div.className = 'vertical-layout';
-    (<VerticalLayout> this.uischema).elements.forEach(element => {
-      const bestRenderer = JsonFormsHolder.rendererService
+    const verticalLayout = <VerticalLayout> this.uischema;
+    if (verticalLayout.elements !== undefined && verticalLayout.elements !== null) {
+      verticalLayout.elements.forEach(element => {
+        const bestRenderer = JsonFormsHolder.rendererService
           .getBestRenderer(element, this.dataSchema, this.dataService);
-      div.appendChild(bestRenderer);
-    });
+        div.appendChild(bestRenderer);
+      });
+    }
     this.appendChild(div);
     return this;
   }
-   dispose(): void {
+  dispose(): void {
     // Do nothing
   }
+  notify(type: RUNTIME_TYPE): void {
```

Comment:
The code of `notify` seems to be the same in all the latter cases. Can this be extracted somehow?